webapp/packages/plugin-sql-editor/src/SqlEditor/useSqlEditor.ts

```diff
@@ -441,6 +441,7 @@ export function useSqlEditor(state: ISqlEditorTabState): ISQLEditorData {
           return this.activeSegment;
         }
 
+        await data.updateParserScripts();
```

Comment:
must be executed before:
```ts
if (!projectId || !connectionId || this.cursor.begin !== this.cursor.end) {
  return this.getSubQuery();
}
```
because it depends on `this.parser.getScriptSegment();` (getSubQuery -> getExecutingQuery -> parser.getScriptSegment)`